packages/script/src/call-test-script/Forc.lock

```diff
@@ -0,0 +1,12 @@
+[[package]]
```

Comment:
You probably don't want to commit the lockfile here, otherwise the stdlib version won't get bumped automatically when you bump the version of `forc` in CI.